app/views/studies/_shared_sync_functions.js.erb

```diff
@@ -1,9 +1,81 @@
 // shared JS functions for sync forms
 
-$('#study-file-<%= study_file.id %>').on('change', '.file-type', function() {
+// Used for sync CSFV
+if (typeof window.SCP.readOnlyToken === 'undefined') {
```

Comment:
why is this `if` needed?   Wouldn't we want the latest version of the token even if it's already defined?

Reply:
I'm actually not sure!  I naively reused the approach from `_study_visualize.html.erb`.   f6ecfa6 removes it.